ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -37,10 +37,17 @@ module Identifier = struct
     monitor_config_file : string option;
   }
 
+  type mxgpu_id = {
+    pdev_id : int; (* PCI id of the PF, not the VFs *)
```

Comment:
Let's explicitly say "device ID", because "PCI ID" is often used for the bus address.